examples/minimal-mdns/AllInterfaceListener.h

```diff
@@ -88,14 +88,24 @@ class AllInterfaces : public mdns::Minimal::ListenIterator
         }
     }
 
+    template <typename T>
+    inline bool MulticastOk(T & iterator)
```

Comment:
This is a lot of code duplication, however the logic of skiping seems generally duplicated, I did not try to centralize as well in this PR.